backend/apis/utils/validation-utils.js

```diff
@@ -26,6 +26,7 @@ exports.nullableNonEmptyString = nullable(_nonEmptyString);
 exports.nonEmptyString = checkDefined(_nonEmptyString);
 
 exports.nullableValidDate = nullable(_validDate);
+exports.nullableValidInteger = nullable(_validInteger);
```

Comment:
This is already defined on line 59 -- maybe a merge conflict problem? https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/5cfaafb6abb009dd2e9d2cc281c025979c9b9203/backend/apis/utils/validation-utils.js#L59

Reply:
Hopefully i fixed this the right way.